style/app.scss

```diff
@@ -685,133 +640,133 @@ a:active
+  border           : solid #d3d4d6;
+  border-width     : 0 1px;
+  margin-top       : 20px;
+  margin-left      : 10px;
 }
 
-#sidebar-button:hover
+.sidebar-dropdown
 {
-  background-color : #cac4d4;
+  padding          : 10px;
 }
 
-#sidebar-icon
+.expanded > .sidebar-button > #sidebar-icon
 {
-  width        : 30px;
-  height       : 50px;
-  padding-top  : 20px;
-  padding-left : 2px;
-  position     : absolute;
-  top          : 40%;
-  left         : 4px;
+  -webkit-transform : rotate(180deg);
+  -moz-transform    : rotate(180deg);
+  -ms-transform     : rotate(180deg);
+  -o-transform      : rotate(180deg);
+  transform         : rotate(180deg);
 }
 
-#focuses-nav, #graph-nav
+div.sidebar.collapsed
 {
-  cursor : pointer;
+  height: 40px;
 }
 
-#sidebar-nav
+.collapsed > .sidebar-dropdown
 {
-  width            : 100%;
-  font-size        : 13px;
-  background-color : #cac4d4;
-  border           : solid 1px #dedede;
-  box-shadow       : 0 2px 2px -1px rgba(0, 0, 0, 0.055);
-  display          : block;
-  overflow         : hidden;
+  display: none;
 }
 
-#sidebar-nav ul
+.sidebar-button
 {
-  width  : 100%;
-  margin : 0 0 0 0;
+  cursor           : pointer;
+  border-radius    : 10px;
+  width            : 30px;
+  height           : 15px;
+  background-color : #5c497e;
+  margin-top       : -5px;
+  margin-left      : 70px;
 }
 
-#sidebar-nav ul li
+.selected-courses
 {
-  list-style-type    : none;
-  display            : inline-block;
-  width              : 48%;
-  -webkit-transition : all 0.2s;
-  -moz-transition    : all 0.2s;
-  -ms-transition     : all 0.2s;
-  -o-transition      : all 0.2s;
-  transition         : all 0.2s;
+  margin-top       : 0px;
+  margin-bottom    : 10px;
+  font-size        : 14px;
+  color            : #946aa8;
 }
 
-#sidebar-nav ul li:hover
+.courses
 {
-  background-color : #5C497E !important;
+  overflow   : auto;
+  max-height : 275px;
+  max-width  : 140px;
+  display    : flex;
+  flex-flow  : column wrap;
 }
 
-#sidebar-nav ul li:hover a
+.course-selection
 {
-  color : white !important;
+  font-size  : 14px;
+  margin-right: auto;
+  order: 2;
 }
 
-#sidebar-nav ul li .nav-section
+.sidebar-button:hover
 {
-  color           : #000000;
-  display         : inline-block;
-  line-height     : 30px;
-  text-align      : center;
-  width           : 95%;
-  text-decoration : none;
+  background-color: #675d79;
 }
 
-#focuses, #graphs
+#sidebar-icon
 {
-  margin-top  : 25px;
-  margin-left : 25px;
-  height      : 86%;
-  overflow-y  : scroll;
+  width : 15px;
+  height: 15px;
 }
 
-.focus
+.fcecount
 {
-  display          : block;
-  cursor           : pointer;
-  font-size        : 20px;
-  border           : solid 1px #000000;
-  text-align       : center;
-  width            : 90%;
+  border-radius    : 10px;
+  line-height      : 40px;
   background-color : #ffffff;
-  border-radius    : 6px;
+  text-align       : center;
+  font-size        : 16px;
+  box-shadow       : 4px 4px 4px #e8e9eb;
```

Comment:
It's a bit strange to have the `box-shadow` set on the `.sidebar` root element when it's expanded, and the `.fcecount` element when collapsed. You can get the same effect by keeping the shadow only on the `.sidebar` element, when both expanded and collapsed.

The same is true for the `border` styling, I think it's better to just have that always be applied to the top `.sidebar` element.